influxql/iterator.gen.go.tmpl

```diff
@@ -1213,9 +1213,27 @@ func (itr *{{$k.name}}Stream{{$v.Name}}Iterator) reduce() ([]{{$v.Name}}Point, e
 // {{$k.name}}{{if ne $k.Name $v.Name}}{{$v.Name}}{{end}}ExprIterator executes a function to modify an existing point
 // for every output of the input iterator.
 type {{$k.name}}{{if ne $k.Name $v.Name}}{{$v.Name}}{{end}}ExprIterator struct {
-	left  *buf{{$k.Name}}Iterator
-	right *buf{{$k.Name}}Iterator
-	fn    {{$k.name}}{{if ne $k.Name $v.Name}}{{$v.Name}}{{end}}ExprFunc
+	left   *buf{{$k.Name}}Iterator
+	right  *buf{{$k.Name}}Iterator
+	fn     {{$k.name}}{{if ne $k.Name $v.Name}}{{$v.Name}}{{end}}ExprFunc
+	points []{{$k.Name}}Point // must be size 2
```

Comment:
Would it be better to have this be an array rather than a slice, so that the compiler can enforce length of 2?